tests/utils.go

```diff
@@ -4661,11 +4661,11 @@ func GenerateRandomMac() (net.HardwareAddr, error) {
 	if err != nil {
 		return nil, err
 	}
-	return net.HardwareAddr(append(prefix, suffix...)), nil
```

Comment:
is this intentional? It's not about parenthesis removal

Reply:
This is a type conversion (wrongly called "cast" in the commit message), from `[]byte` to `net.HardwareAddr`, which is itself defined inside the net module with `type HardwareAddr []byte`.
The type conversion is marked as redundant by GoLand, and indeed appears to be.
However, I can't actually explain why! If HardwareAddr was defined with "type HardwareAddr byte", then a ~~cast~~type conversion from byte to HardwareAddr would be required!
Since I can't fully explain it I'm happy to revert this part...

Reply:
I certainly don't know why, but I can live with this.

Reply:
nit: Since this method returns `net.HardwareAddr` I think that is better to declare the prefix as `prefix := net.HardwareAddr{0x02, 0x00, 0x00} ` and suffix as `suffix := make(net.HardwareAddr, 3)`. I know that it is the same.

Reply:
Agreed, good point, will adjust. Thanks for the review!